kuhub/templates/kuhub/create_blog.html

```diff
@@ -1,18 +1,21 @@
 {% extends "kuhub/base.html" %}
 
 {% block content %}
+
 <div class="col-md-8">
     <div class="card mb-4 mt-4">
         <div class="card-header">
             Create Blog Post
         </div>
         <div class="card-body">
-            <form method="POST">
+            <form method="POST" enctype="multipart/form-data">
```

Comment:
what is entype?

Reply:
Enctype is a thing that used to specifies the way that forms-data should be encoded before sending it to server. To add on, multipart/forms-data is one of a method that is used to include an input file type.